src/watchmaker/utils/urllib/__init__.py

```diff
@@ -4,9 +4,10 @@
                         unicode_literals, with_statement)
 
 # pylint: disable=import-error
-from six.moves.urllib import error, parse, request  # noqa: F401
+from six.moves.urllib import error, parse, request  # noqa F401
 
-from watchmaker.utils.urllib.request_handlers import HAS_BOTO3, S3Handler
+from watchmaker.utils.conditions import HAS_BOTO3
```

Comment:
I think we also talked about moving `conditions` to the top level:

```suggestion
from watchmaker.conditions import HAS_BOTO3
```